lib/immutable-struct.rb

```diff
@@ -72,6 +72,16 @@ def self.new(*attributes,&block)
         end
       end
 
+      define_singleton_method(:coerce) do |value|
```

Comment:
Some backchannel discussion expressed some "meh" about this method name. Other ideas:

- implement `call` so that coercion could be done like `Person.(name: "Jay")`
- name it `from` instead of `coerce`, e.g. `Person.from(person)`

Reply:
I'm :+1: on `from`